syntax = "proto3";

package envoy.config.transport_socket.raw_buffer.v2;

option java_package = "io.envoyproxy.envoy.config.transport_socket.raw_buffer.v2";
option java_outer_classname = "RawBufferProto";
option java_multiple_files = true;

// [#protodoc-title: Raw Buffer]
// [#extension: envoy.transport_sockets.raw_buffer]

// Configuration for raw buffer transport socket.
message RawBuffer {
}
